x/params/types/common_test.go

```diff
@@ -24,36 +23,78 @@ type params struct {
 	BondDenom     string        `json:"bond_denom" yaml:"bond_denom"`
 }
 
-func validateUnbondingTime(i interface{}) error {
-	v, ok := i.(time.Duration)
+func validateUnbondingTime(currentValue, newValue interface{}) error {
+	newUbdTime, ok := newValue.(time.Duration)
 	if !ok {
-		return fmt.Errorf("invalid parameter type: %T", i)
+		return fmt.Errorf("invalid parameter type: %T", newValue)
 	}
 
-	if v < (24 * time.Hour) {
+	if newUbdTime < (24 * time.Hour) {
 		return fmt.Errorf("unbonding time must be at least one day")
 	}
 
+	// ignore error if there is no previous value
+	if currentValue == nil {
+		return nil
+	}
+
+	currentUbdTime, ok := currentValue.(time.Duration)
+	if !ok {
+		return fmt.Errorf("invalid parameter type: %T", currentValue)
```

Comment:
Just a note: if the current value's type was is wrong, it'd be a serious problem, perhaps even worthy of panic. Unsure though